bin/download_state.sh

```diff
@@ -2,6 +2,9 @@
 
 set -e
 
+AVAILABLE_STATES=(409123 408251 407159 404345 379502)
```

Comment:
We should not keep the available states in the installer (otherwise you need to change it everytime a new state is published)

What if we retrieve the list from https://nodes.dusk.network/state/list ?

Reply:
@herr-seppia Yes, that's what I would ideally like, but I have no idea how the current endpoints there work.

If we can do it, that would be cool.